packages/ripple-binary-codec/src/types/uint-16.ts

```diff
@@ -41,7 +46,7 @@ class UInt16 extends UInt {
    * @returns the number represented by this.bytes
    */
   valueOf(): number {
-    return this.bytes.readUInt16BE(0)
+    return parseInt(readUInt16BE(this.bytes, 0))
```

Comment:
Why is this wrapped in a parseInt? (It seems that previously this worked with just the `readUInt16BE` function, which makes me wonder if the implementation in `utils` has the wrong signature?

Reply:
Used to make `BigInt`s else where